lib/device-connections-factory.js

```diff
@@ -40,12 +40,22 @@ class iProxy {
     await status;
   }
 
-  quit () {
+  async quit () {
     if (!this.serverSocket) {
       return;
     }
-    this.serverSocket.close();
-    this.serverSocket = null;
+    return await new B((resolve, reject) => {
```

Comment:
could this not be written more async await style?

```js
try {
   await B.promisify(this.serverSocket.close, {context: this.serverSocket})();
   this.log.info('...');
   this.serverSocket = null;
} catch (err) {
   this.log.error('...');
   throw err;
}
```

I think that's what we tend to do elsewhere (convert to promise rather than wrapping callback in promise)?

Reply:
Cool, I didn't know about such trick